rootfs/scheduler/k8s.py

```diff
@@ -354,87 +360,69 @@ def create(self, name, image, command, **kwargs):
         app_name = kwargs.get('aname', {})
         try:
             self._create_service(name, app_name, app_type)
-        except Exception as e:
+        except:
             self._scale_app(name, 0, app_name)
             self._delete_rc(name, app_name)
-            err = '{} (create): {}'.format(name, e)
-            raise RuntimeError(err)
+            raise
 
     def _get_service(self, name, namespace):
-        con_get = httplib.HTTPConnection(self.url)
-        con_get.request('GET', '/api/'+self.apiversion+'/namespaces/'+namespace+'/services/'+name)
-        resp = con_get.getresponse()
-        reason = resp.reason
-        status = resp.status
-        data = resp.read()
-        con_get.close()
-        if not 200 <= status <= 299:
-            errmsg = "Failed to get Service: {} {} - {}".format(
-                status, reason, data)
-            raise RuntimeError(errmsg)
-        return (status, data, reason)
+        url = self._api("/namespaces/{}/services/{}", namespace, name)
+        resp = self.session.get(url)
+        if not 200 <= resp.status_code <= 299:
+            raise_err(resp, 'get Service "{}" in Namespace "{}"', name, namespace)
+        return resp.status_code, resp.text, resp.reason
 
     def _create_service(self, name, app_name, app_type):
         random.seed(app_name)
         app_id = random.randint(1, 100000)
-        appname = "app-"+str(app_id)
+        appname = "app-" + str(app_id)
```

Comment:
we'll have to fix this up eventually. This should reflect the actual app name, as it's creating a service with the name `app-7216` for an app called `lively-dragster`.